doc/source/tutorial/stats.rst

```diff
@@ -1190,3 +1191,246 @@ because the p-value is very low and the MGC test statistic is relatively high.
 The MGC-map indicates a **strongly nonlinear relationship**. The optimal scale
 in this case is **equivalent to the local scale**, marked by a red spot on the
 map.
+
+Quasi-Monte Carlo
+-----------------
+
+Before talking about Quasi-Monte Carlo (QMC), a quick introduction about Monte
+Carlo (MC). The MC method was called in statistics 'model sampling'. It is
+a method which aims at trying out hypothesis. Once all the hypothesis we
+have defined have been tested, we can compute some statistics such as the
+mean of a given quantity of interest. A scenarios is defined by some variables.
+So, in order to map all scenarios, we use a random number generator.
+
+Generating random numbers is a more complex problem than it sounds. Simple
+Monte Carlo (MC) methods are designed to sample points to be independent and
+identically distributed (IID). But generating multiple sets of random points
+can produce radically different results.
+
+.. plot:: tutorial/stats/plots/qmc_plot_mc.py
+   :align: center
+   :include-source: 0
+
+In both cases in the plot above, points are generated randomly without any
+knowledge about previously drawn numbers. It is clear that some regions of
+the space are left unexplored - which can cause problems in simulations.
+
+A great benefit of MC is that it has known convergence properties.
+Let's look at the mean of the squared sum in 5 dimensions:
+
+.. math::
+
+    f(\mathbf{x}) = \left( \sum_{j=1}^{5}x_j \right)^2,
+
+which has a known mean value, :math:`\mu = 5/3+5(5-1)/4`. Using MC sampling, we
```

Comment:
what kind of distribution do you assume for the x_i?

Reply:
A uniform distribution. Usually when doing convergence analysis you only use uniform distribution as to not bias the results.